deletiontests/azure/azure_delete_test.go

```diff
@@ -25,6 +25,11 @@ func Test_AzureDelete(t *testing.T) {
 
 	ctx := context.Background()
 
+	enableThisTest, exists := os.LookupEnv("TEST_DELETION")
```

Comment:
Instead of making our bash script more complex, I think it's better to skip the test from the test itself. This has also the benefit that the test will show up in the testing report as `Skipped`.